sql/moz-fx-data-shared-prod/activity_stream_bi/impression_stats_by_experiment_v1/metadata.yaml

```diff
@@ -1,5 +1,8 @@
+---
 friendly_name: Impression Stats By Experiment
-description: Representation of tile impression statistics, clustered by experiment_id to allow efficient analysis of individual experiments
+description: Representation of tile impression statistics,
+ clustered by experiment_id to allow efficient analysis of
```

Comment:
Does the line break introduce any new characters?

Reply:
I don't believe so. Why?

Reply:
Some of the tests are breaking lines using the same style, I just wanted to understand the behavior. It looks like trailing whitespaces can be introduced here, which could actually lead to the field being different than it was before the change.